docs/guides/web-ui/alerts.md

```diff
@@ -42,7 +45,7 @@ The **Webhook URL** field is where you can specify a URL to which the alert will
 For now, **Logfire** alerts only send the requests in [Slack format].
 
 ??? tip "Get a Slack webhook URL"
-    To get a Slack webhook URL, follow the instructions in the [Slack documentation](https://api.slack.com/messaging/webhooks).
+To get a Slack webhook URL, follow the instructions in the [Slack documentation](https://api.slack.com/messaging/webhooks).
```

Comment:
this spacing was intentional to make the tip render properly

Reply:
before:

<img width="809" alt="Screenshot 2024-09-26 at 14 32 57" src="https://github.com/user-attachments/assets/3fd50def-1b1b-4a17-9423-8239e125bba9">

after:

<img width="820" alt="Screenshot 2024-09-26 at 14 33 00" src="https://github.com/user-attachments/assets/952ca9ce-bea5-4a3c-bc8a-c56d10f31963">

Reply:
Sorry I think my auto-formatter did that. Was there a reason to make this admonition an accordion @alexmojaki if you want it to be expanded by default? It makes more sense to me to have this be static (not collapsable) in this case, and use the accordion state when you want it collapsed by default.

Reply:
it's not expanded by default. look at https://logfire.pydantic.dev/docs/guides/web-ui/alerts/#create-an-alert and compare with the preview.

Reply:
sorry @alexmojaki I got myself confused looking at your screengrab, I'll fix this up.